src/index.ts

```diff
@@ -799,19 +799,27 @@ const BPLUS: { [ wiringpi: number ]: IRaspiPinInfo } = {
 };
 
 // Initialize the board info
-let procInfo: string;
+let procInfo: string = '';
+let rev: string = '';
+
 if (process.env.RASPI_IO_TEST_MODE) {
   procInfo = 'Revision:a21041';
 } else {
-  procInfo = readFileSync('/proc/cpuinfo').toString();
-}
-const revMatch = procInfo.match(/Revision\s*:\s*(.*)/);
-if (!revMatch) {
-  throw new Error('Unable to parse revision information in /proc/cpuinfo');
+    try {
+      procInfo = readFileSync('/proc/cpuinfo').toString();
+      const revMatch = procInfo.match(/Revision\s*:\s*(.*)/);
+
+      if (!revMatch) {
+          console.warn('Unable to parse revision information in /proc/cpuinfo');
+      } else {
+          [ , rev ] = revMatch;
```

Comment:
Let's rework this to be:

```TypeScript
rev = revMatch[1];
```

I think this is more readable and avoids folks having to know how sparse arrays work (which trips some folks up), and is the same number of characters.

Reply:
done